pyopenetl/operations.py

```diff
@@ -185,7 +185,7 @@ def execute_raw_sql(self, sql: str, use_textual: bool = False):
         print(f"SQL executed in {datetime.datetime.now() - start}")
         return res
 
-    def get_postgres_dtypes_for_temp_table(self, table: str) -> dict:
+    def get_postgres_table_schema(self, table: str) -> dict:
```

Comment:
If you change this function name, make sure you also change all the places it gets called! Try doing a ctrl+f search -- I think there are a few spots where we're still calling it using the old name. I think they might've been pulled in from Sophia's recent change to add the `append_rows_to_table_from_dataframe()` method